scripts/zones/Batallia_Downs/Zone.lua

```diff
@@ -63,7 +63,7 @@ function onInitialize(zone)
     SetVoidwatchNPC(vwnpc);
 
     -- Ahtu
-    SetRespawnTime(17207657, 900, 10800);
+    GetMobByID(17207657):setRespawnTime((math.random(0,4) * 1800) + 900); -- Window is 0-2 hours, since no source shows the window intervals, setting to 30 mins
```

Comment:
@TeoTwawki No, I am not sure. At this point, I am only reformatting what exists already and putting it in the proper windowed format. No documentation shows mobs like Ahtu with a different maint pop window. It could be this was an old insert that was never relevant, but I am unsure what ones should be removed to clean it up.